app/events/db/migrations/db_table_sql_scripts.sql

```diff
@@ -655,3 +655,61 @@ CREATE TABLE IF NOT EXISTS iiq_schema.professional_profile_analytics
 );
 ALTER TABLE iiq_schema.professional_profile_analytics
     OWNER TO iiq;
+
+
+CREATE TABLE IF NOT EXISTS iiq_schema.async_profile_analytics
```

Comment:
don't we have any models for this??

Reply:
No, we are adding the db_mapping and migration script here so that it can be easily integrated by our clients.